releng/org.eclipse.xtext.tycho.tests.parent/pom.xml

```diff
@@ -13,7 +13,7 @@
 
 	<properties>
 		<os-jvm-flags />
-		<memory-settings>-Xmx2G</memory-settings>
+		<memory-settings>-Xmx2G -XX:MaxRAMPercentage=70.0</memory-settings>
```

Comment:
does this work locally?

Reply:
Will check again. From our latest observations it could be that this option is not required. I was experimenting with that.